s3/proto/s3.proto

```diff
@@ -319,3 +320,10 @@ message ListMultipartUploadRequest {
 message ListMultipartUploadResponse {
 	repeated MultipartUploadRecord records = 1;
 }
+
+message StorageTierAndClass {
+	string BackendType = 1;

```

Comment:
The variable name should start with lower case, ie. `s/BackendType/backendType`

Reply:
[Here](https://github.com/opensds/multi-cloud/pull/163/files#diff-fb9cfede8d99166400c22506d697b759R34) should also be changed.

Reply:
Done